### PR TITLE
Unsubscribe

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

**Comment by asood123** [comment]
Can you merge with master?

File: server/controllers/services/email.js [hunk 1]
```diff
@@ -15,18 +17,53 @@ module.exports = (app) => {
...
+  const unsubscribe = (req, res, next) => {
+
+    const type = req.params.type;
+    const email = req.params.email;
+    const slug = req.params.slug;
+
+    const identifier = `${email}.${slug}.${type}.${config.keys.opencollective.resetPasswordSecret}`;
+    const token = crypto.createHash('md5').update(identifier).digest("hex");
+
+    console.log("Unsubscribe ", email, slug, type, token);
```
Comment (asood123) [inline server/controllers/services/email.js]: Is this necessary?

Comment (xdamman) [inline server/controllers/services/email.js]: nope. Just removed it.

File: server/lib/email.js [hunk 2]
```diff
@@ -128,8 +139,8 @@ const generateEmailFromTemplate = (template, recipient, data) => {
  * Deprecated. Should use sendMessageFromActivity() for sending new emails.
  */
 const generateEmailFromTemplateAndSend = (template, recipient, data, options) => {
-  return generateEmailFromTemplate(template, recipient, data)
-    .then(templateString => sendMessage(recipient, getSubject(templateString), getBody(templateString), options));
+  return generateEmailFromTemplate(template, recipient, data, options)
+    .then(templateString => emailLib.sendMessage(recipient, getSubject(templateString), getBody(templateString), options));
```
Comment (asood123) [inline server/lib/email.js]: Where is emailLib coming from? Self-referencing?

**Pull request merged**